src/taipy/gui/gui.py

```diff
@@ -472,7 +472,7 @@ def __get_real_var_name(self, var_name: str) -> t.Tuple[str, str]:
                 raise NameError(f"Can't find matching variable for {var_name} on context: {current_context}")
         return f"{var_name}.{suffix_var_name}" if suffix_var_name else var_name, current_context
 
-    def __call_on_change(self, var_name: str, value: t.Any, on_change: t.Optional[str] = None):
+    def _call_on_change(self, var_name: str, value: t.Any, on_change: t.Optional[str] = None):
         try:
```

Comment:
Will we get an exception in __get_real_var_name with <chart var name>.decimator.... ?

Reply:
I believe we won't get an exception since the table var_name is valid and will get discovered by the get real var name